specs/deneb/validator.md

```diff
@@ -79,29 +78,47 @@ def validate_blobs_and_kzg_commitments(execution_payload: ExecutionPayload,
 
 3. If valid, set `block.body.blob_kzg_commitments = blob_kzg_commitments`.
 
-#### Constructing the `SignedBeaconBlockAndBlobsSidecar`
-To construct a `SignedBeaconBlockAndBlobsSidecar`, a `signed_beacon_block_and_blobs_sidecar` is defined with the necessary context for block and sidecar proposal.
+#### Constructing the `SignedBlobSidecar`s
 
-##### Block
-Set `signed_beacon_block_and_blobs_sidecar.beacon_block = block` where `block` is obtained above.
+To construct a `SignedBlobSidecar`, a `signed_blob_sidecar` is defined with the necessary context for block and sidecar proposal.
 
 ##### Sidecar
-Coupled with block, the corresponding blobs are packaged into a sidecar object for distribution to the network.
 
-Set `signed_beacon_block_and_blobs_sidecar.blobs_sidecar = sidecar` where `sidecar` is obtained from:
+Blobs associated with a block are packaged into sidecar objects for distribution to the network.
+
+Each `sidecar` is obtained from:
 ```python
-def get_blobs_sidecar(block: BeaconBlock, blobs: Sequence[Blob]) -> BlobsSidecar:
-    return BlobsSidecar(
-        beacon_block_root=hash_tree_root(block),
-        beacon_block_slot=block.slot,
-        blobs=blobs,
-        kzg_aggregated_proof=compute_aggregate_kzg_proof(blobs),
-    )
+def get_blob_sidecars(block: BeaconBlock, blobs: Sequence[Blob]) -> Sequence[BlobSidecar]:
+    return [
+        BlobSidecar(
+            block_root=hash_tree_root(block),
+            index=index,
+            slot=block.slot,
+            block_parent_root=block.parent_root,
+            blob=blob,
+            kzg_commitment=block.body.blob_kzg_commitments[index],
+            kzg_proof=compute_kzg_proof(blob),
```

Comment:
`compute_kzg_proof()` does not take a single argument. You probably want to use `compute_blob_kzg_proof()` from https://github.com/ethereum/consensus-specs/pull/3236.

```suggestion
            kzg_proof=compute_blob_kzg_proof(blob),
```

Reply:
a5f61fc173cc4c8bf350bc72945f5cf3dc39bae5